### DISCUSSION TITLE
Whether acceleration can be added to VehicleData?

### DISCUSSION THREAD

**Discussion by radiateboy** [eclipse/mosaic#249] [Ideas]
`
private VehicleData(
            long time, String name, GeoPoint position,
            CartesianPoint projectedPosition, IRoadPosition roadPosition, String routeId,
            double speed, Double heading, double slope, Double longitudinalAcceleration,
            String laneAreaId, double distanceDriven, VehicleStopMode vehicleStopMode,
            VehicleEmissions vehicleEmissions, VehicleConsumptions vehicleConsumptions,
            VehicleSignals vehicleSignals, VehicleSensors vehicleSensors,
            Double brake, Double throttle, DriveDirection driveDirection,
            Object additionalData
    )
`
The acceleration should have lateral acceleration and longitudinal acceleration

**Comment by kschrab** [comment]
Hi, in order to add this to VehicleData, we have to be sure that the traffic simulator (presumably SUMO) provides meaningful values for this. If you can find out and provide us with more information on that, we can discuss deeper if we should add this to VehicleData. The extension should be very simple though. From my understanding, the lateral acceleration is only calculated in SUMO for lane changes if the Sublane Model is activated, which we barely use. In other case this would always be `0`.
